browser.js

```diff
@@ -1,35 +1,40 @@
+var generatePrime = require('./generatePrime');
 var primes = require('./primes.json');
+
 var DH = require('./dh');
-var generatePrime = require('./generatePrime');
 
-exports.DiffieHellmanGroup =
```

Comment:
The reason I put them at the bottom or at least _after_ the declaration is such that JSHINT doesn't complain about variables being used before they are declared.